BUILDING.md

```diff
@@ -0,0 +1,39 @@
+See the [release](https://github.com/rancher/k3s/releases/latest) page for pre-built releases.
+
+The clone will be much faster on this repo if you do
+
+    git clone --depth 1 https://github.com/rancher/k3s.git
+
+This repo includes all of Kubernetes history so `--depth 1` will avoid most of that.
+
+To build the full release binary run `make` and that will create `./dist/artifacts/k3s`.
+
+Optionally to build the binaries without running linting or building docker images:
+```sh
+./scripts/download && ./scripts/build && ./scripts/package-cli
+```
+
+For development, you just need go 1.12 and a proper GOPATH.  To compile the binaries run:
+```bash
+go build -o k3s
+go build -o kubectl ./cmd/kubectl
+go build -o hyperkube ./vendor/k8s.io/kubernetes/cmd/hyperkube
+```
+
+This will create the main executable at `./dist/artifacts` , but it does not include the dependencies like containerd, CNI,
```

Comment:
Is this an issue? Should I be wrapping at about 80 char length to make it terminal-friendly? Or just fix this carriage return?